docs/GETTING_STARTED.md

```diff
@@ -230,7 +233,7 @@ In the example command we arbitrarily are pushing the logs to a file named
 `~/soci-snapshotter-logs`, but please feel free to choose a different location:
 
 ```
-sudo ./soci-snapshotter-grpc&> ~/soci-snapshotter-logs &
+sudo ./soci-snapshotter-grpc 2> ~/soci-snapshotter-errors 1> ~/soci-snapshotter-logs &
```

Comment:
What about keeping both options (everything in one place and stdout is separate from stderr) with clear definitions of which workflow is more feasible for which? 

And with that, you can just write that it is recommended to use one of the options as the default.